README.md

```diff
@@ -107,13 +107,17 @@ For example, a file named `menu.svg` in `~/Sites/sixtwothree.org/images/icons` w
 
 To use an svgeez-generated SVG sprite file, first include the file's contents at the bottom of your HTML page.
 
-In a Rails 4 application:
+In a Rails 5 application:
 
 ```erb
 <body>
     <!-- Your page’s awesome content goes here! -->
 
-    <%= raw Rails.application.assets.find_asset('icons.svg') %>
+    <% if Rails.application.config.assets.compile %>
+      <%= raw Rails.application.assets.find_asset('icons.svg') %>
+    <% else %>
+      <%= raw Rails.application.assets_manifest.find_sources('icons.svg').first %>
+    <% end %>
```

Comment:
According to [this comment](https://github.com/rails/sprockets-rails/issues/237#issuecomment-231948159) (and confirmed by my own testing), `find_sources` will work in _both_ development and production. So in a Rails 5 application, this block could be shortened to:

```
<%= raw Rails.application.assets_manifest.find_sources('icons.svg').first %>
```